ip_localities.py

```diff
@@ -0,0 +1,79 @@
+from arcgis import GIS
+import pandas as pd
+import math
```

Comment:
PEP257 style imports are native, third party, and then local and alphabetized between : https://www.python.org/dev/peps/pep-0257/

This would be :
```
import math

from arcgis import GIS
import pandas as pd
```